packages/components/src/placeholder/test/index.tsx

```diff
@@ -74,9 +74,12 @@ describe( 'Placeholder', () => {
 			);
 			expect( placeholderInstructions ).not.toBeInTheDocument();
 
-			// Test for empty fieldset.
-			const placeholderFieldset =
-				within( placeholder ).getByRole( 'group' );
+			// Test for empty content. When the content is empty,
+			// the only way to query the div is with `querySelector`
+			// eslint-disable-next-line testing-library/no-node-access
```

Comment:
Not sure why unit tests were unhappy about this. I guess it changed recently.

Reply:
Mhh, there isn't an obvious better choice. Maybe @tyxla can think of something here?

Reply:
Thanks for the ping @ciampo! 

Nope, if we remove the `fieldset` in favor of a `div` (which I agree it makes sense), there's no other way to query for the empty div. There are other DOM traversing alternatives, but this one is the most precise. 

Therefore, since I agree with removing the `fieldset` in favor of a `div` in this use case, I'm fine with the ignore. Making an element inaccessible intentionally is the ideal case for this type of ESLint ignore.